scheduler/src/cook/scheduler/constraints.clj

```diff
@@ -90,6 +90,13 @@
   (let [previous-hosts (job->previous-hosts-to-avoid job)]
     (->novel-host-constraint job previous-hosts)))
 
+(defn get-gpu-model-requested
+  "Get GPU model requested from job environment or use default GPU model on pool"
+  [gpu-count gpu-model-in-env pool-name]
+  (when (pos? gpu-count)
+    (or gpu-model-in-env
+        (util/match-based-on-pool-name (config/valid-gpu-models) pool-name :default-model))))
```

Comment:
Worth a comment:
; lookup the GPU model from the pool defaults defined in config.edn

Reply:
sure, added